### DISCUSSION TITLE
Arduino Nano to STM32-Blackpill connection mapping

### DISCUSSION THREAD

**Discussion by EngMarc** [Zer0-bit/gaggiuino#342] [Q&A]
So, definitely love the project and a big shout out to all involved - Thank You! I'm an avid reuse/repurpose person so this is right down the alley of what I want to do with some great working hardware.

My question:
1) theres the diagram GC-Conections-Diagram.png  which is a great resource for pin connections to the Arduino Nano.  The STM32 board is very different and noted to be used for advanced functionality. I don't see a pin connection mapping for the STM32. So trying to figure out if a person was to go down the path of using the STM32 board, how would the connections be made?   E.g. is there a STM32 diagram somewhere?

![GC-CONNECTIONS-DIAGRAM](https://user-images.githubusercontent.com/48916817/204253875-a3f809d5-24b7-4d22-a372-fabd0d68b12f.png)

**Comment by Zer0-bit** [comment]
Where are you looking though ?

**Reply by EngMarc** [reply]
Zero, what do you mean where am I looking through?
If you mean in what directory - schematics from the github repository.

**Reply by Zer0-bit** [reply]
No i mean if you've seen the stuff on https://gaggiuino.github.io/

**Reply by Zer0-bit** [reply]
Mainly the "STM32 Upgrade" part

**Reply by EngMarc** [reply]
Yes, I have read through that but didn't see a diagram so was wondering if there was one for the STM32.

**Reply by Zer0-bit** [reply]
Errrr ...yes ? Under "STM32 UPGRADE" you can find a table with the wires switching  and a diagram?

**Reply by EngMarc** [reply]
ok so that small diagram there replaces the old Arduino nano diagram?
I've copied it to my directory of my working project so I'll have it. 
I'm planning to do the STM32 project first off since it has more functionality that I may be able to use on my Platinum project. Starting out simple and moving to more complex 👯‍♂️

**Reply by Zer0-bit** [reply]
Yeah i saw your other post under discussions, i wish you luck with the project it's definitely going to be frustrating at points but very rewarding when/if you pull it off.

**Reply by Zer0-bit** [reply]
As a sidenote you might consider joining the discord server and create a thread under **#install-help-experimental-zone**, as you can then bump ideas against more heads

**Reply by EngMarc** [reply]
I tried to join the discord group with the link on the web page but it said it was expired.

**Reply by Zer0-bit** [reply]
Can't be expired, the invite has no time limit.
What webpage you referring to ?

**Reply by EngMarc** [reply]
this one:
https://gaggiuino.github.io/#/learning/learning-sources?id=resources

**Reply by EngMarc** [reply]
here's the screen shot when I try to use the http link in the add channel area on discord.

**Comment by EngMarc** [comment]
<img width="468" alt="Screen Shot 2022-11-28 at 19 33 35" src="https://user-images.githubusercontent.com/48916817/204409388-4528b597-c742-4d23-b37a-459b350a6307.png">

**Reply by Zer0-bit** [reply]
![image](https://user-images.githubusercontent.com/42692077/204555021-713f96e3-c14a-43c2-b611-92204fb60a71.png)

**Reply by EngMarc** [reply]
That doesn't tell me how to add the gaggiuino discord channel on my discord app/machine. 
Should I just search for the channel by name or what?

**Reply by EngMarc** [reply]
I opened the file and only the bottom link to discord works. 
Thanks
